akka-stream/src/main/scala/akka/stream/scaladsl/Sink.scala

```diff
@@ -31,6 +31,28 @@ final class Sink[-In, +Mat](private[stream] override val module: Module)
 
   override val shape: SinkShape[In] = module.shape.asInstanceOf[SinkShape[In]]
 
+  def contramap[In2](f: In2 ⇒ In): Sink[In2, Mat] = Flow.fromFunction(f).toMat(this)(Keep.right)
+
+  /**
+   * Connect this `Sink` to a `Flow`.
+   * This function is equivalent to [[Flow#to]].
+   *
+   * The materialized value of the combined [[Sink]] will be the materialized
+   * value of the given flow (ignoring the current Sink’s value), use
+   * [[Sink#prependMat[Mat2* prependMat]] if a different strategy is needed.
```

Comment:
These links do not work in JavaDoc after genjavadoc translation, please just use backticks for a code snippet.